tests/Hakyll/Web/Pandoc/Biblio/Tests.hs

```diff
@@ -64,3 +65,36 @@ goldenTest01 =
             cleanTestEnv
 
             return output)
+
+goldenTest02 :: TestTree
+goldenTest02 =
+    goldenVsString
+        "biblio02"
+        (goldenTestsDataDir </> "biblio01.golden")
+        (do
+            -- Code lifted from https://github.com/jaspervdj/hakyll-citeproc-example.
+            logger <- Logger.new Logger.Error
+            let config = testConfiguration { providerDirectory = goldenTestsDataDir }
+            _ <- run RunModeNormal config logger $ do
+                let myPandocBiblioCompiler = do
+                        csl <- load "chicago.csl"
+                        bib <- load "refs.yaml"
```

Comment:
Please include this file in `Extra-source-files` section in hakyll.cabal (note it's sorted alphabetically, please keep the ordering).